app/views/password_resets/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<main class="page-content">
+  <div class="wrapper">
+    <h1>Enter the email associated with your account:</h1>
+    <%= render partial: 'layouts/flash' %>
```

Comment:
I think putting the flash partial in `shared/_flash.html.erb` is more conventional.